source/common/matcher/solo_matcher.cc

```diff
@@ -120,16 +119,9 @@ class PathMatcherImpl : public BaseMatcherImpl {
 class RegexMatcherImpl : public BaseMatcherImpl {
 public:
   RegexMatcherImpl(const RouteMatch &match) : BaseMatcherImpl(match) {
-    if (match.path_specifier_case() ==
```

Comment:
This is the main change that I'm unsure is safe. Envoy removed all of these deprecated fields, do we need to continue to support this case?

Reply:
This looks reasonable to me.. we only use safeRegex in the CP for gloo edge. https://github.com/solo-io/solo-projects/blob/b6f702f52a180818b83f966aebb81a30d34d608b/projects/gloo/pkg/plugins/advanced_http/plugin.go#L238